### ISSUE TITLE
Cursor Visual Bug 

### ISSUE THREAD

**Issue by Martajnez** [ionutbortis/gnome-bedtime-mode#26]
Hi! I use Pop os and I come across this bug while bedtime mode is enabled.

https://photos.google.com/share/AF1QipP3i45xJtnKgBWxxSfRRobGeMvlc5LsEO1bcT-D9uc6bJUnpgIcXb-1v1UNCJfGVA/photo/AF1QipPdddvn9q0lGZix3ZOLbInxUxuSg-Wc0T13kU1w?key=N1M0SkVBOVdRbmwxNVI0YWk1ZFExN3ZSZGxmeFlR

**Comment by ionutbortis** [comment]
Hi @Martajnez, 

Can you please give me more details on your desktop environment?
PopOS version, Gnome version, other extensions that might be of interest?

Also, how did you configure the Bedtime Mode extension? Which color preset are you using?

**Comment by Martajnez** [comment]
Sure, here you are
PopOS: 20.04 LTS
GNOME: 3.36.8

Grayscale 100% intensity with schedule. Intensity chance by scrolling turned off 

To be honest I am new in the  Linux world and I don't see any malicious extension that could cause the problem. 😟

**Comment by ionutbortis** [comment]
@Martajnez 

I tried unsuccessfully to reproduce the issue on:
- A PopOS vm using boxes
- A PopOS live usb stick on my home laptop ( Asus VivoBook with AMD cpus and gpu)

Also enabled Night Light during my tests, but with no success...

I'm not sure what's causing that mouse issue on your system. Maybe it's something related to your video card drivers.

Can you test this behavior using the TintAll extension? https://extensions.gnome.org/extension/1471/tint-all/

Just disable/remove the Bedtime Mode extension, install the TintAll extension and circle between the effects by clicking the new icon from the top bar. Scrolling over the icon will increase/decrease the effect intensity.

Do you have the same issue when using the TintAll extension?

**Comment by Martajnez** [comment]
Hello, thanks for your response. I tested the TintAll extension, unfortunately result were the same. Although I tried to reinstall my Pop OS. On latest build 21.04 everything seems working without any problems. I think that it had to be an accident. Sorry for wasting your time, I am really thankful for your extension.

**Comment by ionutbortis** [comment]
Great news!

I suspected that it might be something related to video card drivers. I also encountered some artifacts on Ubuntu and Fedora, but they are rather inconsistent in reproducing. I believe that Gnome color effects implementation might lack in certain areas.

For example, I'm not able to apply the color effect to the mouse pointer. I spoke with the guys from the Gnome Extensions channel and there is no way to do that by using an extension.

I'm happy that everything works fine now on your side and I'm also glad that I got a chance to try out PopOS :sweat_smile:
